variables.tf

```diff
@@ -1317,3 +1317,47 @@ variable "confluence_s3_attachments_storage" {
   type        = bool
   default     = false
 }
+
+# test deployment configuration
+
+variable "start_test_deployment" {
+  description = "Whether to start DCAPT Jmeter and Selenium deployment"
```

Comment:
Should we say something like "Deploy necessary resources to start DCAPT testing"? So it is potentially flexible in future.